### ISSUE TITLE
Still Maintaining?

### ISSUE THREAD

**Issue by buddhi-deep** [nick92/tranqil#64]
As the title,cuz i found that most of the way to install it is useless now

**Comment by nick92** [comment]
Not really, don't have the time unfortunately but do love the app so will hopefully be able to do some work on it at some point
